solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -129,17 +132,17 @@ contract LiquidityPoolV1Converter is LiquidityPoolConverter {
         assert(amount < targetReserveBalance);
 
         // ensure that the input amount was already deposited
-        if (_sourceToken == ETH_RESERVE_ADDRESS)
+        if (address(_sourceToken) == ETH_RESERVE_ADDRESS)

```

Comment:
If we use `ETH_RESERVE_ADDRESS` as `IERC20Token` everywhere in this file, then you may want to consider declaring it as such to begin with.